sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/coders/Coder.java

```diff
@@ -17,14 +17,14 @@
  */
```

Comment:
Can you modify the import order in `checkstyle.xml`?

Right now, it says that `com.google` comes first, and then the rest alphabetically.

    <property name="groups" value="com.google,android,com,io,Jama,junit,net,org,sun,java,javax"/>

Can we do `org.apache.beam,com.google,<the rest alphabetically>`? I think this will make imports much more readable. Once the renaming is complete, we should remove `com.google` from the list.

If so, local project-imports come first, then an empty line, then the rest.